packages/manager/modules/pci/src/projects/project/contacts/contacts.controller.js

```diff
@@ -1,157 +1,59 @@
-import indexOf from 'lodash/indexOf';
-
-export default /* @ngInject */ function (
-  OvhApiCloud,
-  OvhApiCloudProjectServiceInfos,
-  OvhApiMe,
-  $stateParams,
-  CucCloudMessage,
-  CucControllerHelper,
-  $translate,
-  PCI_REDIRECT_URLS,
-  $window,
-  coreConfig,
-) {
-  const self = this;
-  const serviceName = $stateParams.projectId;
-
-  self.model = {
-    owner: '',
-    billing: '',
-    isAdmin: false, // do we have admin rights?
-    isUSorCA: false, // true if user country is USA or Canada
-  };
-
-  // admin & billing contact form
-  self.contactFormData = {
-    owner: '',
-    billing: '',
-  };
-
-  // add user right form
-  self.addRightFormData = {
-    contact: '', // either Nic or Email depending on user country
-    type: 'readOnly',
-  };
-
-  // reference to the right the user want to delete
-  self.removeRight = {
-    accountId: null,
-  };
-
-  // toggle display of form inputs
-  self.toggle = {
-    owner: false,
-    billing: false,
-    addUser: false,
-  };
-
-  // table data
-  self.data = {
-    rights: [],
-  };
-
-  // spinners ...
-  self.loader = false;
-
-  /* ==================================================
-      * Initialization
-      */
-
-  function initContact() {
-    return OvhApiCloudProjectServiceInfos.v6().get({
-      serviceName,
-    }).$promise.then((infos) => {
-      self.model.owner = infos.contactAdmin;
-      self.contactFormData.owner = infos.contactAdmin;
-      self.model.billing = infos.contactBilling;
-      self.contactFormData.billing = infos.contactBilling;
-      return OvhApiMe.v6().get().$promise.then((me) => {
-        if (me.nichandle === infos.contactAdmin) {
-          self.model.isAdmin = true;
-        }
-        if (me.country) {
-          // check if the user country is USA or Canada, in this case we display
-          // email instead of NIC handle
-          self.model.isUSorCA = indexOf(['US', 'CA'], me.country.toUpperCase()) >= 0;
-        }
-      });
-    });
+export default class {
+  constructor(
+    $translate,
+    $window,
+    coreConfig,
+    OvhApiCloud,
+    CucCloudMessage,
+    PCI_REDIRECT_URLS,
+  ) {
+    this.$translate = $translate;
+    this.$window = $window;
+    this.coreConfig = coreConfig;
+    this.OvhApiCloud = OvhApiCloud;
+    this.CucCloudMessage = CucCloudMessage;
+    this.PCI_REDIRECT_URLS = PCI_REDIRECT_URLS;
   }
 
-  self.init = function init() {
-    self.getRights();
-    initContact();
-  };
+  $onInit() {
+    this.model = {
+      owner: null,
+      billing: null,
+      isAdmin: false,
+    };
+    this.model.owner = this.projectInfo.contactAdmin;
```

Comment:
You could directly fill the model 
```suggestion
    this.model= {
     owner: this.projectInfo.contactAdmin,
     billing: this.projectInfo.contactBilling,
     isAdmin: this.user.nichandle === this.projectInfo.contactAdmin
     }
```